### ISSUE TITLE
EOS-7237 Implementing HA DB

### ISSUE THREAD

**Issue by prathameshrodi** [Seagate/cortx-py-utils#48]
No description provided.

**Comment by cortx-admin** [comment]
> Donald R Bloyer commented in Jira Server:

[~932497] [~744292]  If this is merged what is holding things up?

**Comment by cortx-admin** [comment]
> Rahul Tripathi commented in Jira Server:

This issue belongs to cortx-py-utils repo, which is a different repo from cortx-utils. Foundation team is not maintaining this repo.

What I know is, that we stopped maintaining cortx-py-utils repo and needed code was moved from cortx-py-utils to cortx-utils around a couple of years ago.

*We need to archive [https://github.com/Seagate/cortx-py-utils]. It was supposed to be deleted long back. It was merged into cortx-utils*

CC: [~747271], [~932497]
